singularity-eos/eos/eos_gruneisen.hpp

```diff
@@ -175,6 +175,8 @@ PORTABLE_INLINE_FUNCTION Real Gruneisen::ComputeRhoMax(const Real s1, const Real
   };
 
   // Comparisons to zero are actually made to
+  // TODO(JMM): Should this become robust::EPS()? They differ by a
+  // factor of 10.
   static constexpr Real EPS = std::numeric_limits<Real>::epsilon();
```

Comment:
I don't think the factor of 10 will make a difference. It looks like I used the EPS parameter just for zero-valued polynomial coefficients. Since the behavior of the polynomial will be very similar for a non-zero coefficient if that coefficient is very small, I think the analysis is still valid.

Reply:
:+1: ok. It's simpler to use robust::eps so I'll try and make that change and see if tests till pass.